app/javascript/components/journey/SolutionFilter.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
+import { usePanel } from '../../hooks/use-panel'
+import { GraphicalIcon } from '../common'
+
+type SolutionFilterOption = {
+  value: string
+  title: string
+}
+
+const OPTIONS: SolutionFilterOption[] = [
+  {
+    value: 'oop',
+    title: 'OOP',
+  },
+  {
+    value: 'functional',
+    title: 'Functional',
+  },
+]
+
+export const SolutionFilter = ({
+  value,
+  setFilter,
+}: {
+  value: string[]
```

Comment:
Should this be plural?

```suggestion
  values: string[]
```